ext/random/tests/02_engine/all_serialize_native.phpt

```diff
@@ -1,27 +1,35 @@
 --TEST--
-Random: Engine: serialize: native
+Random: Engine: Serialization of native engines must preserve the sequence
 --FILE--
 <?php
 
+use Random\Engine\Mt19937;
+use Random\Engine\PcgOneseq128XslRr64;
+use Random\Engine\Xoshiro256StarStar;
+
 $engines = [];
-$engines[] = new \Random\Engine\Mt19937(1234);
-$engines[] = new \Random\Engine\PcgOneseq128XslRr64(1234);
-$engines[] = new \Random\Engine\Xoshiro256StarStar(1234);
+$engines[] = new Mt19937(1234);
```

Comment:
I was thinking of a form like the first PR, where the implemented class name is defined in a separate file and iterated over. e.g.) `engines.inc`.
https://github.com/zeriyoshi/php-src/blob/ed42ae2c2fe30ad9e608b6e693cd3ef3c60f1d85/ext/random/tests/number_generator/common.inc

Reply:
I find this not as readable, because one needs to keep the contents of the include file in the head, instead of simply reading the test from top to bottom.